### ISSUE TITLE
Great Project - Looking forward to future releases (Dataset deletion doesn't work)

### ISSUE THREAD

**Issue by burny0202** [jonmatifa/zfsmanager#25]
Unable to delete dataset.

Great work man

**Comment by jonmatifa** [comment]
Thanks burny0202! Can you give me more info about the dataset you're unable to delete?

**Comment by burny0202** [comment]
Hi there. I did it manually following my post, it was a pool and dataset that I had imported over from nas4free. I was able to destroy under root in terminal and was logged in webmin as root also

**Comment by jonmatifa** [comment]
What happened when you tried to delete those in zfs manager? Did it give you any kind of error?

**Comment by burny0202** [comment]
No; it just did nothing. I've had this on the snapshot tab in the following days too. Perhaps I will try another browser next time it happens. I'm on quantum at the moment
